config/error_reporting.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+return [
+    // Enable error reporting
+    'enable' => (bool) env('STACKDRIVER_ENABLED', false),
+    'logName' => env('STACKDRIVER_LOGGING_NAME', 'error-log'),
+    'serviceId' => env('STACKDRIVER_ERROR_REPORTING_SERVICE_ID', env('APP_NAME')),
+    'versionId' => env('STACKDRIVER_ERROR_REPORTING_VERSION_ID', '1.0.0'),
+    'LoggingClient' => [
```

Comment:
As we mentioned in person it could be good to take 5mins to check if there is a way to make this logging appears as though its from a k8s container and add all the appropriate metadata to make it good; if not then no worries!

Reply:
Cool, let's just go ahead and merge this so we're unblocked on the laravel 10 update. I think we should then (not immediately) revisit our whole strategy and consider "just" logging to stdout/stderr similar to mediawiki

Reply:
> I think we should then (not immediately) revisit our whole strategy and consider "just" logging to stdout/stderr similar to mediawiki

sounds interesting, happy to talk about and improve this